adyenv6core/resources/adyenv6core-items.xml

```diff
@@ -285,6 +313,10 @@
                     <description>Number of Installments</description>
                     <persistence type="property"/>
                 </attribute>
+                <attribute qualifier="adyenTerminalId" type="java.lang.String">
+                    <description>Terminal ID</description>
+                    <persistence type="property"/>
+                </attribute>
```

Comment:
we do not need this right as we are using the API to retrieve the terminals ?

Reply:
This is for storing terminal information in database.